### PR TITLE
2 create architecture

### PR DESCRIPTION
**Pull request by LucasTesnier**
Create the basic server architecture and a prebase for @31Nathan 
(Moving the work of @TomDUVAL-MAHE )

### CODE REVIEW

**Review by 31Nathan — DISMISSED** [review]
Seems good to me, I will add the prisma schema after

**Review by 31Nathan — DISMISSED** [review]

**Comment by osvegn** [comment]
Need to update 'tests/index.test.js' with the import '../index.js' to the new filename @LucasTesnier

File: database/dump.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-- MySQL Script generated by MySQL Workbench
+-- Wed Jan 18 14:59:22 2023
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+ALTER USER 'root' IDENTIFIED WITH mysql_native_password BY 'password';
```
Comment (osvegn) [inline database/dump.sql:6]: Unsecured

File: sources/server_app.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  } else {
+    next()
+  }
+})
+
+app.get('/', (req, res) => {
+  res.send('Hello World')
+})
+
+app.get('/about.json', async (req, res) => {
```
Comment (osvegn) [inline sources/server_app.js]: why this route is usefull ?

**Review by osvegn — CHANGES_REQUESTED** [review]
Unsecured password defined
config route seems useless

**Review by osvegn — APPROVED** [review]

**Review by 31Nathan — APPROVED** [review]

**Pull request merged**